pymc/distributions/shape_utils.py

```diff
@@ -490,85 +465,50 @@ def convert_size(size: Size) -> Optional[StrongSize]:
         raise ValueError(
             f"The `size` parameter must be a tuple, TensorVariable, int or list. Actual: {type(size)}"
         )
-    if isinstance(size, tuple) and Ellipsis in size:
-        raise ValueError(f"The `size` parameter cannot contain an Ellipsis. Actual: {size}")
 
     return size
 
 
-def resize_from_dims(dims: WeakDims, ndim_implied: int, model) -> Tuple[StrongSize, StrongDims]:
-    """Determines a potential resize shape from a `dims` tuple.
+def shape_from_dims(dims: Dims, shape_implied: Sequence[TensorVariable], model) -> StrongShape:
+    """Determines shape from a `dims` tuple.
 
     Parameters
     ----------
     dims : array-like
-        A vector of dimension names, None or Ellipsis.
-    ndim_implied : int
-        Number of RV dimensions that were implied from its inputs alone.
+        A vector of dimension names or None.
+    shape_implied : tensor_like of int
+        Shape of RV implied from its inputs alone.
     model : pm.Model
         The current model on stack.
 
     Returns
     -------
-    resize_shape : array-like
-        Shape of new dimensions that should be prepended.
     dims : tuple of (str or None)
-        Names or None for all dimensions after resizing.
+        Names or None for all RV dimensions.
     """
-    if Ellipsis in dims:
-        # Auto-complete the dims tuple to the full length.
-        # We don't have a way to know the names of implied
-        # dimensions, so they will be `None`.
-        dims = (*dims[:-1], *[None] * ndim_implied)
     sdims = cast(StrongDims, dims)
```

Comment:
Signature violation: The previous `dims: WeakDims` parameter was already a sequence, but with `dims: Dims` the new signature allows for `dims="bla"`.
This is not accounted for in the implementation; the `cast(StrongDims, dims)` is not safe yet.

Reply:
Removed the cast, now that the signature of `shape_from_dims` requires `StrongDims`